src/Microsoft.ML/Runtime/EntryPoints/CrossValidationMacro.cs

```diff
@@ -188,7 +203,10 @@ public sealed class CombinedOutput
                 var args = new TrainTestMacro.Arguments
                 {
                     Nodes = new JArray(graph.Select(n => n.ToJson()).ToArray()),
-                    TransformModel = null
+                    TransformModel = null,
+                    LabelColumn = input.LabelColumn,
+                    GroupColumn = input.GroupColumn.IsExplicit ? input.GroupColumn : Optional<string>.Implicit(DefaultColumnNames.GroupId),
+                    WeightColumn = input.WeightColumn.IsExplicit ? input.WeightColumn : Optional<string>.Implicit(DefaultColumnNames.Weight)
```

Comment:
Does the second branch of the logic for GroupColumn again define the same default from Arguments? (I could be misunderstanding the code.) If it does repeat, might it be better to have it just use the value already assigned to the input arguments?